autogen/agentchat/contrib/gpt_assistant_agent.py

```diff
@@ -93,6 +90,7 @@ def __init__(
                     instructions,
                     openai_assistant_cfg.get("tools", []),
                 )
+                instructions = candidate_assistants[0].instructions
```

Comment:
Why is this line needed? Line 174 updates the system message.
I do think that line 173 and 174 can be merged by setting the right system message in line 173 directly, unless
@IANTHEREAL suggests otherwise.

Reply:
Sorry for the late reply. At lines 71 and 73, we will remove the model from `openai_client_cfg` caused by openai assistant api design. Please confirm if this will cause an error with `super().__init__.` I remember that `ConversableAgent._validate_llm_config` checks the configuration of the model. If it doesn’t cause an error now, please follow @sonichi suggestion.